users.js

```diff
@@ -966,6 +964,8 @@ class User {
 			}
 		}
 		this.confirmed = '';
+		this.setGroup(' ');
+		this.updateIdentity();
```

Comment:
Again, this is the wrong place for `updateIdentity`.

Identity update should happen after a user is locked.